js/src/views/ParityBar/parityBar.css

```diff
@@ -95,20 +94,25 @@ $modalZ: 10001;
   }
 }
 
+.accountsSection {
+  padding: 0 1em;
+  width: 920px;
```

Comment:
So it was looking very strange when you switch from 2 accounts displayed to 3 accounts (try it out). The layout would be totally different. Here it is more consistent whatever the number of accounts to show, and it is consistent with the width of the Signer.